### PR TITLE
feat(dangerJS): extended conventional commits message

### PR DESCRIPTION
**Pull request by gmoraleda**
**JIRA**
n/a

**Summary**
In the mobile team, we have extended the explanation of conventional commits to clarify when the PR doesn't match the criteria. 
It helps new joiners to understand the requirement better.

(For some reason, the format was out of sync with the one specified in `prettierrc`).

### CODE REVIEW

**Comment by capmo-jarvis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please ensure the package version has been bumped in accordance with [Semantic Versioning](https://semver.org/)
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **This PR is not assigned to you and/or someone else**
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  
**The JIRA-ticket has not been filled out**
In case, there is no JIRA-ticket, please replace the example URL by 'n/a'.

  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  PR title *MUST* follow conventional commits and include a JIRA-ID (optional)./n<type>[optional scope]: <description>/nE.g./n`feat: [KEY-123] Add new login button` or `fix(login): [KEY-123] Fix error message`/n/nThe most common types are:/n- **`feat`**: A new feature/n- **`fix`**: A bug fix/n- **`docs`**: Documentation only changes/n- **`style`**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)/n- **`refactor`**: A code change that neither fixes a bug nor adds a feature/n- **`perf`**: A code change that improves performance/n- **`test`**: Adding missing tests or correcting existing tests/n- **`chore`**: Changes to the build process or auxiliary tools and libraries such as documentation generation/n/nSee [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for more info./n/n🚓 Be a good commitizen
Details:
* subject-case - subject must not be sentence-case, start-case, pascal-case, upper-case

  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against ddd85a1235e2d6b787b3eeb5f4fb4c5c44ec2507
</p>

**Review by tanveer-sid — APPROVED** [review]
LGTM!

File: dangerjs/title.js [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
+  '- **`feat`**: A new feature',
+  '- **`fix`**: A bug fix',
+  '- **`docs`**: Documentation only changes',
+  '- **`style`**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)',
+  '- **`refactor`**: A code change that neither fixes a bug nor adds a feature',
+  '- **`perf`**: A code change that improves performance',
+  '- **`test`**: Adding missing tests or correcting existing tests',
+  '- **`chore`**: Changes to the build process or auxiliary tools and libraries such as documentation generation',
+  '',
+  'See [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for more info.',
```
Comment (constantin-melniciuc) [inline dangerjs/title.js:20]: `https://www.conventionalcommits.org/en/` would redirect to the latest version. This would help keep it up to date.

**Review by constantin-melniciuc — APPROVED** [review]
Great suggestions! 🚀 
Minor nitpick :)

**Pull request merged**